### ISSUE TITLE
Extension broke on update. Had to reinstall. Schema error.

### ISSUE THREAD

**Issue by Bananaman** [Leleat/Tiling-Assistant#199]
Hello! :)

I was suddenly getting an error that Tiling Assistant could not load due to a "gsettings schema" error and it being unable to find the key "transparent-on-moving".

> JS Error: GSettings key transparent-on-moving not found in schema org.gnome.shell.extensions.tiling-assistant

I investigated the source files and found that extension.js tried to read the "transparent-on-moving" variable, and then I looked in the schema folder of the extension and noticed that it indeed didn't have that variable in there. Likewise dconf didn't show that variable.

I was not seeing any available updates for the extension.

So I uninstalled Tiling Assistant, reset/deleted the settings via dconf-editor and then reinstalled the extension via Extension Manager (the flatpak app).

It now loads properly and I noticed that it has a new extension.js without those calls (no "transparent-on-moving" in it anymore).

I am curious:

1. Was a broken version uploaded to GNOME extensions website?
2. Or do you think that my system somehow only partially upgraded the extension? I use a mix of GNOME Extensions app and Extension Manager flatpak to upgrade extensions so I can't guess which one I used.

Thanks again for Tiling Assistant. It is my favorite extension of all!

**Comment by Leleat** [comment]
> Thanks again for Tiling Assistant. It is my favorite extension of all!

Thanks 😄

> transparent-on-moving

I don't recall ever having such a setting. There is also no reference to it in the `extension.js` file from EGO (after manually downloading and unzipping the package). 

Is it possible that another extension was overwriting the file to add some feature that's related to moving windows around?

**Comment by Bananaman** [comment]
@Leleat OH GOD. As you suggested, I did a ripgreo for "transparent-on-moving" across all my extensions...

...and I have no idea how but I recognize the lines now and can see that somehow the Transparent Window Moving's extension.js update ended up in both Tiling Assistant and its own folder. But the references to "transparent-on-moving" were on different line numbers (there were 4 references in Tiling Assistant and 2 in Transparent Window Moving).

I checked to see if TWM had any references to writing into Tiling Assistant but there were none.

Maybe a bug in Extension Manager which may have broken it on update.

At least that means it is totally not your fault! So I am closing this. 😂
